src/test/java/homeworks/RenamePlaylistHmw21.java

```diff
@@ -0,0 +1,38 @@
+package homeworks;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RenamePlaylistHmw21 extends MyBaseTest {
+    @Test
+    public void renamePlaylist()  {
+
+        login();
+        doubleClickChoosePlaylist();
+        enterPlaylistName();
+        Assert.assertTrue(doesPlaylistExist());
+    }
+
+    private boolean doesPlaylistExist() {
+        WebElement playlistElement = driver.findElement(By.xpath("//a[text()='Sucre']"));
+        return playlistElement.isDisplayed();
+
+    }
+    private void enterPlaylistName() {
+        WebElement playlistInputField = driver.findElement(By.xpath("//*[contains(text(),'Edit')]"));
+        playlistInputField.sendKeys((Keys.chord(Keys.CONTROL, "a", Keys.BACK_SPACE)));
+        playlistInputField.sendKeys("Sucre");
+        wait.until(ExpectedConditions.textToBePresentInElement(playlistInputField,"Sucre"));
+        playlistInputField.sendKeys(Keys.ENTER);
+    }
+    private void doubleClickChoosePlaylist() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)")));
+        WebElement element = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        actions.doubleClick(element).perform();
+    }
+
```

Comment:
**Keep It Simple!** 

For example 

```
public class MyLoginTests extends MyBaseTest {

  String expectResultUrl = "https://bbb.testpro.io/";

  @Test(priority = 1)
  public void LoginEmptyEmailPasswordTest() {
    clickSubmitBtn();
    Assert.assertEquals(driver.getCurrentUrl(), expectResultUrl);
  }

  @Test(priority = 2)
  public void LoginValidEmailPasswordTest() {
    login();
    Assert.assertTrue(isAvatarIconPresent());

  }

  private boolean isAvatarIconPresent() {
    WebElement avatarIconElement = driver.findElement(By.xpath("//img[contains(@alt,'Avatar of')]"));
    return avatarIconElement.isDisplayed();
  }

}
```

![image](https://user-images.githubusercontent.com/104761482/203240452-2f92444b-51d0-4345-b67c-ffa84cd77f57.png)